PointOfSaleSystem.API/Controllers/EmployeeController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using PointOfSaleSystem.API.Models;
+using PointOfSaleSystem.API.RequestBodies.Company;
+using PointOfSaleSystem.API.Services.Interfaces;
+using PointOfSaleSystem.API.RequestBodies.Employee;
+
+namespace PointOfSaleSystem.API.Controllers
+{
+    [ApiController]
+    [Route("api/")]
+    public class EmployeeController : ControllerBase
+    {
+        private readonly IEmployeeService _employeeService;
+
+        public EmployeeController(IEmployeeService employeeService)
+        {
+            _employeeService = employeeService;
+        }
+
+        [HttpPost("employee")]
+        public async Task<IActionResult> CreateEmployee(Employee employee)
```

Comment:
When trying to call the CreateEmployee() endpoint in swagger, it request a lot of unnecessary parameters (for example EstablishmentService' whole body). I did the same mistake in CreateCompany() so mb. I guess we should add another request body in the RequestBodies/Employee/ directory, for example -  CreateEmployeeRequest.cs and inside it will contain everything that is in the BaseModel.cs and Employee.cs classes.